### DISCUSSION TITLE
Asus Rog.

### DISCUSSION THREAD

**Discussion by ooemir23** [mathoudebine/turing-smart-screen-python#216] [Themes]
Hello,

I'm sorry for my bad english. Screenshot of the theme I prepared for Xuanfang B. I'm sharing this because I don't know how to add the theme here completely.
![1](https://user-images.githubusercontent.com/74644887/236343238-11919b66-f43e-4617-a2ac-6186b550ad4a.jpg)
![R7 7800x3d](https://user-images.githubusercontent.com/74644887/236343274-20e22203-15c7-4161-b05f-a1d4ec6268cc.png)

**Comment by Paradoxum** [comment]
change the height (up/down) of the background image, it is on top of the data elements
